packages/components/src/popover/style.scss

```diff
@@ -183,6 +183,14 @@ $arrow-size: 8px;
 	height: 100%;
 }
 
+.components-popover {
+	opacity: 0;
+}
+
+.components-popover[data-x-axis][data-y-axis] {
+	opacity: 1;
+}
```

Comment:
I'm not sure I understand why the opacity trick here

Reply:
It's more or less the same as before, where for `master` it's setting `visibility: hidden;` for as long as the position hasn't been calculated. The calculation can only happen when the DOM element has already rendered, but before the calculation happens we don't want the DOM element to show.

Reply:
I'll leave an inline comment.